net/Socket.cpp

```diff
@@ -555,6 +555,33 @@ void StreamSocket::dumpState(std::ostream& os)
         Util::dumpHex(os, "\t\toutBuffer:\n", "\t\t", _outBuffer);
 }
 
+void StreamSocket::sendError(int errorCode,
+               const std::string& body, const std::string& extraHeader, bool shutdownAfter)
+{
+
+    std::ostringstream oss;
+    oss << "HTTP/1.1 " << errorCode << "\r\n"
+        << "Date: " << Util::getHttpTimeNow() << "\r\n"
+        << "User-Agent: " << WOPI_AGENT_STRING << "\r\n"
```

Comment:
The first two << operators in each line here are unnecessary.

Reply:
IMO removing them from these two (now three) lines makes the whole thing look inconsistent, doesn't it?